packages/firebase_ml/lib/src/firebase_model_download_conditions.dart

```diff
@@ -0,0 +1,21 @@
+part of firebase_ml;
```

Comment:
I think at some point we will have to do it. It's a fairly easy process, and I wasn't sure how to name the directory (because technically firebase model manager manages other things like FirebaseTranslateRemoteModel too). If we agree on the name right now, I can create a separate folder. Something like "firebase_model_management" so the other one would be "firebase_vision"? Or we can drop the word "firebase" altogether.

These are the conventions if you are interested: https://dart.dev/tools/pub/package-layout

Reply:
I think it's fine to move to a separate directory later if we find it helpful.